shopping-cart/deploy/scripts/common/installers.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash 
+
+installOC() {
+    if [ -z $(command -v oc) -a $TRAVIS ]
+    then
+        (
+            mkdir tmp 
+            cd tmp 
+            echo "Installing oc 3.11"
+            curl -OL https://github.com/openshift/origin/releases/download/v3.11.0/openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit.tar.gz
+            echo "4b0f07428ba854174c58d2e38287e5402964c9a9355f6c359d1242efd0990da3  openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit.tar.gz" | sha256sum -c
+            tar --strip 1 -xvzf openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit.tar.gz && chmod +x oc && sudo cp oc /usr/local/bin/ && rm oc
+        )
+        rm -rf tmp/
+    else 
+        echo "! oc already installed."
+    fi
+    oc version && echo "oc CLI installed successfully"
+}
+
+installKustomize() {   
```

Comment:
Kubectl 1.14+ have `kustomize` out-of-the-box. We played with him. OC until haven't?

Reply:
Interesting. 
TBH, I added this code here but it's not used and we'll probably remove it. We've found other ways to manage the YAML files without kustomize. This is a left-over from lagom/lagom-scala-openshift-smoketests

Reply:
> We've found other ways to manage the YAML files without kustomize

We too 😄 
Now for Lagom 1.5.X we migrated from LBO to HELM charts.

Reply:
Anyway, I think `oc` `3.11` uses `kubectl` `1.11` :-/

Reply:
I realised over the weekend that `kustomize` being part of `kubectl` will require us changing some [docs](https://www.lagomframework.com/documentation/current/java/Migration15.html#Docker-images-and-deployment-specs). Thanks for sharing @ihostage ! ❤️